src/data.js

```diff
@@ -0,0 +1,98 @@
+const START_DATE = new Date(2019, 1, 1);
+const END_DATE = new Date(2020, 1, 1);
+const MAX_COST = 300;
+const MIN_COST = 10;
+const MAX_OFFERS = 2;
+const MAX_PHOTOS = 10;
+const MAX_SENTENCES = 3;
+const MAX_CARDS = 13;
+const TYPE_EVENTS = {
+  'Taxi': `🚕`,
+  'Bus': `🚌`,
+  'Train': `🚂`,
+  'Ship': `🛳️`,
+  'Transport': `🚊`,
+  'Drive': `🚗`,
+  'Flight': `✈️`,
+  'Sightseeing': `🏛️`,
+  'Check-in': `🏨`,
+  'Restaurant': `🍴`,
+};
+const OFFERS = [`Add luggage`, `Switch to comfort class`, `Add meal`, `Choose seats`];
+const DESC = [
+  `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras aliquet varius magna, non porta ligula feugiat eget. Fusce tristique felis at fermentum pharetra. Aliquam id orci ut lectus varius viverra.`,
+  `Nullam nunc ex, convallis sed finibus eget, sollicitudin eget ante. Phasellus eros mauris, condimentum sed nibh vitae, sodales efficitur ipsum.`,
+  `Sed blandit, eros vel aliquam faucibus, purus ex euismod diam, eu luctus nunc ante ut dui.`,
+  `Sed sed nisi sed augue convallis suscipit in sed felis.`, `Aliquam erat volutpat. Nunc fermentum tortor ac porta dapibus. In rutrum ac purus sit amet tempus.`,
+];
+const CITIES = [`Amsterdam`, `Geneva`, `Chamonix`, `London`, `Berlin`, `Vienna`, `Paris`, `Manchester`];
+const NAMES = [`Taxi to Airport`, `Flight to Geneva`, `Drive to Chamonix`, `Check into a hotel`];
+
+const getRandomInt = (max = 1, min = 0) => {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  if (max < min || max < 0 || min < 0) {
+    return 0;
+  }
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+const getRandomDate = (start, end) => {
+  return new Date(getRandomInt(end, start));
+};
+
+const doMixOfArray = (array, elements) => {
```

Comment:
```suggestion
const doMixOfArray = (array, count) => {
```
`elements` подразумевает что ты передаешь массив элементов, а не их количество